daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -194,15 +197,21 @@ object Value {
   type VersionedContractInstance = transaction.Versioned[ContractInstance]
 
   object VersionedContractInstance {
-    def apply(template: Identifier, arg: VersionedValue): VersionedContractInstance =
-      arg.map(ContractInstance(template, _))
+    def apply(
+        packageName: Option[Ref.PackageName] = None,
+        template: Identifier,
+        arg: VersionedValue,
+    ): VersionedContractInstance =
+      arg.map(ContractInstance(packageName, template, _))
 
+    @deprecated("use the version with 3 argument", since = "2.9.0")
```

Comment:
Is it still used anywhere? I see that you've replaced a bunch of occurrences in this PR.

Reply:
Maybe in canton.